### PR TITLE
yum: correct update command

### PR DESCRIPTION
**Pull request by apm1467**
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

The original command `yum update` upgrades installed packages. `yum check-update` is the correct command which synchronizes list of packages and versions available.

### CODE REVIEW

**Comment by navarroaxel** [comment]
@apm1467 Please sign the `license/cla`. Also, I don't agree with the example description: `This should be run first, before running subsequent yum commands`.

I can run `yum upgade` without run `yum update` or `yum check-update`.

**Comment by apm1467** [comment]
@navarroaxel You are right. It is normally not necessary to run `yum check-update` at all. People get the habit from `brew` or `apt-get` that they should always run `update` before doing other things. This is not how `yum` works.

Do you think it is appropriate to remove this whole command?

**Comment by navarroaxel** [comment]
@apm1467 Indeed not, the `yum` page has 6 examples and the limit is 8. So, you can fix the example description and add the `check-update` example if you want. Also you can rearrange the examples if you think another order is better to learn this command.

And, you can add pages for the subcommands like `yum-update.md` to explore the subcommand options with more detail.

File: pages/linux/yum.md [hunk 1]
```diff
@@ -22,6 +18,10 @@
 
 `yum remove {{package}}`
 
+- Get a list of available updates for installed packages:
```
Comment (navarroaxel) [inline pages/linux/yum.md:21]:
```suggestion
- Display available updates for installed packages:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages/linux/yum.md [hunk 2]
```diff
@@ -2,10 +2,6 @@
 
 > Package management utility for RHEL, Fedora, and CentOS (for older versions).
```
Comment (navarroaxel) [inline pages/linux/yum.md:3]: Could you add a link to a web man page? I found this:
```suggestion
> Package management utility for RHEL, Fedora, and CentOS (for older versions).
> More information: <https://man7.org/linux/man-pages/man8/yum.8.html>.
```
Do you know a better one?

**Review by navarroaxel — CHANGES_REQUESTED** [review]
1 comment. Let me know if you're done with the editions, or if you're planning more.

File: pages/linux/yum.md (hunk 2 above)
Comment (apm1467) [reply pages/linux/yum.md:3]: I looked for similar pages like [`apt`](https://github.com/tldr-pages/tldr/blob/cd3a80bffbd3f6c26c39e9513b5e072f58c9ddfa/pages/linux/apt.md) and [`apk`](https://github.com/tldr-pages/tldr/blob/5ef55e6fc44ddb75672469a9403e49702bc27753/pages/linux/apk.md), and they do not have a web man page link. I think the one you found is decent.

**Comment by apm1467** [comment]
@navarroaxel Thank you very much for suggestions and help. I'm done with editing.

**Review by navarroaxel — APPROVED** [review]
LGTM, thanks @apm1467.

**Review by sbrl — APPROVED** [review]
Thanks, @apm1467! That's interesting that `yum` doesn't need you to update the repo metadata first.

**Pull request merged**